model-engine/model_engine_server/inference/tensorrt-llm/triton_model_repo/postprocessing/1/model.py

```diff
@@ -147,10 +176,25 @@ def finalize(self):
         """
         print("Cleaning up...")
 
-    def _postprocessing(self, tokens_batch):
+    def _postprocessing(self, tokens_batch, sequence_lengths):
         outputs = []
-        for beam_tokens in tokens_batch:
-            for tokens in beam_tokens:
-                output = self.tokenizer.decode(tokens)
+        for batch_idx, beam_tokens in enumerate(tokens_batch):
+            for beam_idx, tokens in enumerate(beam_tokens):
+                seq_len = sequence_lengths[batch_idx][beam_idx]
+                output = self.tokenizer.decode(
+                    tokens[:seq_len], skip_special_tokens=self.skip_special_tokens
```

Comment:
why do we restrict to [:seq_len], what are in `tokens` that outside of `seq_len`?

Reply:
It's present in the original code https://github.com/triton-inference-server/tensorrtllm_backend/blob/v0.8.0/all_models/inflight_batcher_llm/postprocessing/1/model.py#L192